### PR TITLE
OCC-1182, OCC-468, FMRPA-35

### PR DESCRIPTION
**Pull request by JalenSowell47**

### CODE REVIEW

File: app/views/user_mailer/_fmr_email_header.haml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 %h1{style: 'text-align:left'}
-  =["You have booked the following trip(s)", Config.application_title.present? ? "via #{Config.application_title}" : nil].compact.join(" ") + ":"
+    = translate('api_v1.emails.user_trip_header.opener')
```
Comment (drewteter) [inline app/views/user_mailer/_fmr_email_header.haml:2]: This has one too many indents. The tests can't read the haml file.

File: spec/mailers/user_mailer_spec.rb
Comment (drewteter) [inline spec/mailers/user_mailer_spec.rb:1]: Thank you for including a test.

File: 1.18.10_translation_migration.json
Comment (drewteter) [inline 1.18.10_translation_migration.json:1]: Should this file be removed? It looks like you added the text to the en.yml file

Comment (JalenSowell47) [reply 1.18.10_translation_migration.json:1]: This is part of what I wanted to discuss with you. Rebecca and I looked at it last week and weren't sure the en.yml file was working how it should or was effective or actually added new translations to the admin area like it was supposed to, so we settled on creating a JSON file when we want to add translations to 1-click.

**Pull request merged**